science/gildas/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           active_variants 1.1
 
 name                gildas
 version             202107a
-revision            1
+revision            2
 set my_version      [string tolower [clock format [clock scan 2000-[string range ${version} 4 5]-10] -format %b]][string range ${version} 2 3][string range ${version} 6 end]
 set my_machine      [if {[string match *arm* ${os.arch}]} {list arm64} {list x86_64}]
```

Comment:
Based on this line, it appears that this port intends to be used only on arm64 or x86_64 systems. If that's so, could we add the line:
```
supported_archs arm64 x86_64
```
That way an appropriate error message will be emitted early on systems that are not arm64 nor x86_64 (such as i386, ppc, or ppc64).

Reply:
Ok, I will add this.